AnkiDroid/src/main/java/com/ichi2/anki/dialogs/DatabaseErrorDialog.java

```diff
@@ -223,7 +223,16 @@ public MaterialDialog onCreateDialog(Bundle savedInstanceState) {
                                         return true;
                                     });
                 }
-                return builder.show();
+                MaterialDialog materialDialog = builder.build();
+                materialDialog.setOnKeyListener((dialog, keyCode, event) -> {
+                    if (keyCode == KeyEvent.KEYCODE_BACK) {
+                        Timber.i("catch");
```

Comment:
Without a bit more specifics on where the log came from, it will be hard to trace, suggest 👇 

```suggestion
                        Timber.i("DIALOG_RESTORE_BACKUP caught hardware back button");
```